www/manager/templates/html/Feed/Media/FileUpload.tpl.php

```diff
@@ -34,7 +34,17 @@
                     <div class="dcf-ratio dcf-ratio-16x9 mh-upload-box" id="mh_upload_media">
                         <div class="dcf-ratio-child dcf-d-flex dcf-flex-col dcf-ai-center dcf-jc-center dcf-txt-center">
                             <span>Add Media</span>
-                            <p class="dcf-mb-0">.mp4 or .mp3<br>(Maximum file size: <?php echo UNL_MediaHub_Controller::$max_upload_mb; ?>mb)</p>
+                            <p class="dcf-mb-0">
+                                <?php if ($user->canTranscode()): ?>
+                                    mp4, .mov, or .mp3
+                                    <br>
+                                    (Maximum file size: <?php echo UNL_MediaHub_Controller::$max_upload_mb * 10; ?>mb or ~10gb)
```

Comment:
if $max_upload_mb is changed to something other than 1024 then "~10gb" isn't going to be accurate any more